nodeup/pkg/model/kube_apiserver.go

```diff
@@ -318,6 +352,42 @@ func (b *KubeAPIServerBuilder) buildPod() (*v1.Pod, error) {
 
 	pod.Spec.Containers = append(pod.Spec.Containers, *container)
 
+	if b.Cluster.Spec.EncryptionConfig != nil && *b.Cluster.Spec.EncryptionConfig {
+		sideCar := &v1.Container{
+			Name:  "kms-plugin",
+			Image: "sethpollack/kmsplugin:latest",
+			Args: []string{
+				"-listen",
+				"/var/run/kmsplugin/socket.sock",
+				"-key",
+				keyName,
+			},
+			LivenessProbe: &v1.Probe{
+				Handler: v1.Handler{
+					HTTPGet: probeAction,
+				},
+				InitialDelaySeconds: 15,
+				TimeoutSeconds:      15,
+			},
+			Ports: []v1.ContainerPort{
+				{
+					Name:          "local",
+					ContainerPort: 8080,
+					HostPort:      8080,
```

Comment:
Does this expose the port on the instance?  I'm not sure we want to do that - do we?